source/numerics.tex

```diff
@@ -17093,10 +17098,7 @@
 
 \pnum
 \remarks
-The calls to \tcode{gen} are unsequenced with respect to each other.
-Vectorization-unsafe\iref{algorithms.parallel.defns} standard library functions
-may not be invoked by \tcode{gen}.
-\tcode{gen} is invoked exactly once for each $i$.
+\tcode{gen} is invoked exactly once for each $i$, in increasing order of $i$.
```

Comment:
This wording is hand-wavy; is that ok because it's just a remark? Same for this wording added elsewhere...

Reply:
What is hand-wavy about it? And what would you say instead?

Reply:
What is "each $i$"? The closest wording prior to this paper is "if $N = 0$ or if for each $i$ where $0 \le i < N$ in increasing order"

Reply:
The preceding paragraph says:
> Initializes the $i^\text{th}$ element with
`static_cast<value_type>(gen(integral_constant<simd-size-type, i>()))` for all $i$ in the range of [0,`size()`).

Doesn't that sufficiently explain what "each $i$" means? (And there's no $N = 0$ case to consider here.)